packages/api-client/package.json

```diff
@@ -61,7 +61,6 @@
   "devDependencies": {
     "@scalar/api-client-proxy": "workspace:*",
     "@scalar/echo-server": "workspace:*",
-    "@scalar/use-codemirror": "^0.7.17",
```

Comment:
Why is that removed? 🤔 We’re importing from it in @scalar/api-client.

Reply:
it was duplicated, still in there